tox.ini

```diff
@@ -0,0 +1,59 @@
+# Tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py36, py37, py38, py39, pypy3
+skip_missing_interpreters = True
+skipsdist = True
+
+[testenv]
+deps = -rrequirements.txt
+       -rrequirements-dev.txt
+commands = flake8
+           pip install test/dist/upcloud_api_dev.tar.gz
```

Comment:
This is needed while upcloud python api is not updated with https://github.com/UpCloudLtd/upcloud-python-api/pull/82 and while the changes do not reflect when installing the api with pip. After it is updated the `dist` directory should be deleted and  `requirements.txt` file should be appended with the latest upcloud python api version.

Reply:
While running tox flake8 and safety checks are also ran with every supported python version.

Reply:
When running `tox` flake8 and safety checks are also ran (not just tests).